CMakeLists.txt

```diff
@@ -825,6 +825,9 @@ if(HAVE_LIBWS2_32)
 elseif(HAVE_LIBSOCKET)
   set(CMAKE_REQUIRED_LIBRARIES socket)
 endif()
+if(CMAKE_SYSTEM_NAME STREQUAL Windows)
```

Comment:
`HAVE_IF_NAMETOINDEX` should probably be considered too?

Reply:
Sure, that is certainly reasonable.  The thing is that the code doesn't conditionally build that in the nmake version.  You can't really do conditional linkage in the VS projects either.  At the end of the day, the linking semantics on PE/COFF will drop the link anyways.